src/main/steps/common/common.content.ts

```diff
@@ -10,7 +10,7 @@ const en = {
   applyForAdoption: 'Apply for a Service',
   applyForDissolution: 'Apply for a Service',
   feedback:
-    'This is a new service – your <a class="govuk-link" aria-label="Feedback link, This will open a new tab. You’ll need to return to this tab and continue with your application within 60 mins so you don’t lose your progress." href="#" target="_blank">feedback</a> will help us to improve it.',
+    '<b>Help us improve this service</b><br>This is a new service.Help us for others to improve it by  <a class="govuk-link" aria-label="Feedback link," href="https://www.smartsurvey.co.uk/s/Specials_Feedback/?pageurl=X" target="_blank">giving your feedback</a>',
```

Comment:
The `pageurl` parameter needs to be the actual page rather than "X".  Also needs a space after "service."

Reply:
this has been changed

Reply:
This has been changed to the exit survey which I don't think is correct.  Still needs to be the feedback survey, but with the pageurl as a dynamic parameter.

Reply:
Fixed now